packages/data-requests/src/helpers.rs

```diff
@@ -0,0 +1,139 @@
+use std::collections::HashMap;
+
+use crate::state::DataRequestInputs;
+use crate::utils::hash_data_request;
+use common::msg::PostDataRequestArgs;
+use common::state::{DataRequest, Reveal};
+use common::types::Hash;
+use common::types::{Bytes, Commitment};
+
+use sha3::Digest;
+use sha3::Keccak256;
+
+pub fn calculate_dr_id_and_args(
```

Comment:
I believe this function and `construct_dr()` below need to be marked`#[cfg(test)]` otherwise they will compile to the contract. It would be cleanest to do this in `lib.rs` like you're already doing for the staking contract